app/flatpak-builtins-utils.c

```diff
@@ -1088,3 +1088,50 @@ as_store_find_app (AsStore *store,
 
   return NULL;
 }
+
+static void
+print_line_wrapped (int cols, const char *line)
+{
+  g_auto(GStrv) words = g_strsplit (line, " ", 0);
+  int i;
+  int col = 0;
+
+  for (i = 0; words[i]; i++)
+    {
+       int len = g_utf8_strlen (words[i], -1);
+       int space = col > 0;
+
+       if (col + space + len >= cols)
+         {
+           g_print ("\n%s", words[i]);
+           col = len;
+         }
+       else
+         {
+           g_print ("%*s%s", space, "", words[i]);
+           col = col + space + len;
+         }
```

Comment:
This doesn't seem to correctly handle the case where the first word is wider than cols. We should always print at least one word per line.